terraform/deployments/govuk-publishing-platform/signon_oauth_apps.tf

```diff
@@ -0,0 +1,91 @@
+locals {
+  oauth_applications = [
+    module.oauth_applications["authenticating_proxy"].app_data,
+    module.oauth_applications["content_store"].app_data,
+    module.oauth_applications["draft_content_store"].app_data,
+    module.oauth_applications["publisher"].app_data,
+    module.oauth_applications["publishing_api"].app_data,
+    module.oauth_applications["router_api"].app_data,
+    module.oauth_applications["draft_router_api"].app_data,
+  ]
```

Comment:
Is it feasible to generate this with a [for expression](https://www.terraform.io/docs/language/expressions/for.html)?

Reply:
Ah thank you so much! I thought I was missing something here.